aspnetcore/host-and-deploy/iis/development-time-iis-support.md

```diff
@@ -1,102 +1,108 @@
 ---
 # Development-time IIS support in Visual Studio for ASP.NET Core
 
 By [Sourabh Shirhatti](https://twitter.com/sshirhatti) and [Luke Latham](https://github.com/guardrex)
 
-This article describes [Visual Studio](https://www.visualstudio.com/vs/) support for debugging ASP.NET Core apps running behind IIS on Windows Server. This topic walks through enabling this feature and setting up a project.
+This article describes [Visual Studio](https://www.visualstudio.com/vs/) support for debugging ASP.NET Core apps running with IIS on Windows Server. This topic walks through enabling this scenario and setting up a project.
 
 ## Prerequisites
 
-* [Visual Studio for Windows](https://www.microsoft.com/net/download/windows)
+* [Visual Studio for Windows](https://visualstudio.microsoft.com/downloads/)
 * **ASP.NET and web development** workload
 * **.NET Core cross-platform development** workload
-* X.509 security certificate
+* X.509 security certificate (for HTTPS support)
 
 ## Enable IIS
 
-1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
-1. Select the **Internet Information Services** check box.
-
-![Windows Features showing Internet Information Services check box checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
+1. In Windows, navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+1. Select the **Internet Information Services** check box. Select **OK**.
 
 The IIS installation may require a system restart.
 
 ## Configure IIS
 
 IIS must have a website configured with the following:
 
-* A host name that matches the app's launch profile URL host name.
-* Binding for port 443 with an assigned certificate.
-
-For example, the **Host name** for an added website is set to "localhost" (the launch profile will also use "localhost" later in this topic). The port is set to "443" (HTTPS). The **IIS Express Development Certificate** is assigned to the website, but any valid certificate works:
-
-![Add Website window in IIS showing the binding set for localhost on port 443 with a certificate assigned.](development-time-iis-support/_static/add-website-window.png)
-
-If the IIS installation already has a **Default Web Site** with a host name that matches the app's launch profile URL host name:
-
-* Add a port binding for port 443 (HTTPS).
-* Assign a valid certificate to the website.
+* **Host name** &ndash; Typically, the **Default Web Site** is used with a **Host name** of `localhost`. However, any valid IIS website with a unique host name works.
+* **Site Binding**
+  * For apps that require HTTPS, create a binding to port 443 with a certificate. Typically, the **IIS Express Development Certificate** is used, but any valid certificate works.
```

Comment:
> create a binding to port 443 with a certificate. 
How do you create a binding port with a cert? 
I like previous better
The port is set to "443" (HTTPS).
Do you really want to add **binding**? To
For HTTP, use port 80 or create port 80. Binding is probably the correct way to describe this, but I haven't seen it used very often.

Reply:
Can't cross-link this to IIS docs. They're ... mmm ... their docs are "*very interesting*". :smile:

I'll touch it on the next commit given that we're saying to use the Default Web Site; but truly, this is a *can of worms*. The reader has to both play with IIS and look at the IIS topics. We can't cover all of the angles here. I'll add the IIS docs link to the *Additional resources*.